storybook/src/components/ProgressList/ProgressList.stories.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import type { Meta, StoryObj } from '@storybook/react-vite'
+
+import { ProgressList, UnorderedList } from '@amsterdam/design-system-react/src'
+
+const meta = {
+  title: 'Progress List',
+  component: ProgressList,
+  args: {
+    children: [
+      <ProgressList.Step heading="2025" key={0}>
+        <UnorderedList>
+          <UnorderedList.Item>Start aanleg nieuwe openbare ruimte rondom HEROS (LEBO blok C).</UnorderedList.Item>
+          <UnorderedList.Item>Oplevering lammetjesfontein.</UnorderedList.Item>
```

Comment:
Please add some variation to the content – maybe one with a paragraph, one with a list, and one with both? We can use the `example*` functions here.

An example with a `Paragraph` seems to indicate that we could use a `small` space between the heading and the body – so that requires moving the padding to that latter div as well.